lib/licensee/project_files/license_file.rb

```diff
@@ -3,6 +3,32 @@ class Project
     class LicenseFile < Licensee::Project::File
       include Licensee::ContentHelper
 
+      # List of extensions to give preference to
+      EXTENSIONS = %w(md markdown txt).freeze
+      EXTENSIONS_REGEX = Regexp.union(EXTENSIONS)
+
+      # Regex to match any extension
+      ANY_EXTENSION_REGEX = /[^.]+/
```

Comment:
Ok, I see that it's not just about the format of the file, but actual preferences for it. In that case, I would perhaps use `PREFERRED_EXTENSION_REGEX`, to match `ANY_EXTENSION_REGEX`.